audmetric/core/api.py

```diff
@@ -1186,28 +1186,74 @@ def weighted_confusion_error(
 def word_error_rate(
     truth: Sequence[Sequence[str]],
     prediction: Sequence[Sequence[str]],
+    *,
+    norm: str = "truth",
 ) -> float:
     r"""Word error rate based on edit distance.
 
+    The word error rate is computed
+    by aggregating the normalized edit distances
+    of each (truth, prediction)-pair
+    and averaging the aggregated score
+    by the number of pairs.
+
+    The normalized edit distance
+    of each (truth, prediction)-pair is computed
+    as the edit distance divided by a normalization factor n.
+    This represents the average editing cost per sequence item.
+
+    The value of n depends on the norm parameter:
```

Comment:
I would group it to the last paragraph and remove `:`.

```suggestion
    This represents the average editing cost per sequence item.
    The value of n depends on the ``norm`` parameter.
```

Reply:
Without comitting this suggestion, this is exactly how it looks.